bundles/org.openhab.binding.lutron/README.md

```diff
@@ -368,6 +369,33 @@ Thing configuration file example:
 Thing shade libraryshade [ integrationId=33]
 ```
 
+### Blinds [**Beta**]
+
+Each Lutron Sivoia QS Venetian Blind or Horizontal Sheer Blind is controlled by a **blind** thing.
+Besides `integrationId`, it requires that the parameter `type` be set to either "Venetian" for venetian blinds or "Sheer" for horizontal sheer blinds.
+There is no default.
+If discovery is used, the `type` parameter will automatically be correctly set when the **blind** thing is created.
```

Comment:
I'm no native speaker, but this sentence doesn't seem right: suggestion -> ... will be set automatically when ...

Reply:
I think it may actually be correct, but it does sound awkward. I'll reword it.